packages/frontend-shared/cypress/support/e2e.ts

```diff
@@ -235,8 +235,10 @@ function openGlobalMode (options: OpenGlobalModeOptions = {}) {
   })
 }
 
-function openProject (projectName: ProjectFixtureDir, argv: string[] = []) {
-  if (!fixtureDirs.includes(projectName)) {
+type WithPrefix<T extends string> = `${T}${string}`;
```

Comment:
Same here, this allows us to open Cypress from **within** a project to support monorepo testing